Apps/Contoso.Android.Puppet/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.microsoft.appcenter.xamarin.puppet" android:versionCode="61" android:versionName="2.3.1-SNAPSHOT" android:installLocation="auto">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.microsoft.appcenter.xamarin.puppet" android:versionCode="61" android:versionName="2.4.1-SNAPSHOT" android:installLocation="auto">
```

Comment:
That would be 2.4.0 and we usually update version in a separate PR. Any chance we can revert those?

Reply:
Sure, will do